### DISCUSSION TITLE
What is the meaning of RAUC acronym?

### DISCUSSION THREAD

**Discussion by DerWatz** [rauc/rauc#967] [Q&A] [answered]
Hello, 
sorry 4 this stupid question, but could not find any answer for it. 
C ist for Container (I have heard)? What are the others for?  
R
A
U
C  Container?

**Comment by a3f** [comment, marked as answer]
From the [README](https://github.com/rauc/rauc#rauc---robust-auto-update-controller) title:

> RAUC - Robust Auto-Update Controller

**Reply by DerWatz** [reply]
... I feel so blind. Thank You for Your help. 
Makes no sense 4 me, but what do me noob know ;-). 
So is not an container but a controller ... So I know for the future.

**Reply by ejoerns** [reply]
'Robust' makes sense for me since we do atomic updates and care much for stability, backwards-compatibility and also security,
'Auto-Update' refers to the automatism behind finding the right partitions / slots as update target (less to automatic triggering)
'Controller' refers to the RAUC service on the target that controls the update.

However, 'Container' would not fit any of the current concepts or methods used in RAUC - except for that the bundle somehow is a 'container' of images. But this would still confuse with how the term 'container'  is commonly used and associated nowadays.

I would agree that it is not a perfect name, but those are hard to find, anyway. This could be the reason why it is not massively propagated in contrast to the abbreviation ('RAUC'). The latter is at least quite unique for googling it :smirk:
